lessons/beginners/hello-world/index.md

```diff
@@ -54,11 +54,6 @@ Pak a zadej tento příkaz:
 (venv)$ python ahoj.py
 ```
 
-> [note] Poznámka pro Windows a starší Python
-> V nečeských Windows s Pythonem 3.5 či nižším bude třeba před
-> programem spustit `chcp 1250`, jinak bude program píšící české
-> znaky končit chybou `UnicodeEncodeError`.
```

Comment:
to už teda není potřeba?

Reply:
Ne, předpokládáme Python 3.6, který má f-strings a funguje na Windows :)